config/observability/openshift/telemetry.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: telemetry.istio.io/v1alpha1
 kind: Telemetry
 metadata:
   name: namespace-metrics
-  namespace: gateway-system
+  namespace: istio-system
```

Comment:
Telemetry resource is a istio thing not a envoy resource. Envoy has there own version but that can be post v1